docs/src/main/asciidoc/firestore.adoc

```diff
@@ -60,6 +60,22 @@ The Spring Boot starter for Google Cloud Firestore provides the following config
 |=========================================================
 
 
+==== Supported types
+Firestore  object conversion supports the following types:
+
+* Long
+* Integer
+* Double
+* Float
+* String
+* Boolean
+* Character
+* Map
+* List
+* Enum
+
+POJOs containing fields of these types are also supported.
```

Comment:
Move this sentence above the list please to have it all in one sentence, which I already kind of did above.